final-project/final-project/Controllers/ShelterOneController.cs

```diff
@@ -4,81 +4,85 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
 using final_project.Domain.Models;
+using final_project.Service.Services;
 
 namespace final_project.WebUI.Controllers
 {
     public class ShelterOneController : Controller
     {
-        private List<ShelterOne> Animals = new List<ShelterOne>
+        private readonly IShelterService __shelterService;
```

Comment:
even though the convention uses a single `_` underscore there is no harm on double `__`. Just stay consistent across your code